backend/tests/repository/test_github_provider.py

```
@@ -64,17 +69,17 @@ def test_validate_token_with_network_error(self, mocker):
         # Mock decrypt_git_token to return the token as-is
         mocker.patch("shared.utils.crypto.decrypt_git_token", return_value="any_token")
         mocker.patch("shared.utils.crypto.is_token_encrypted", return_value=True)
-        
+
         # Mock requests.get to raise RequestException
         mock_get = mocker.patch(
```

Ruff (0.14.11)

Local variable `mock_get` is assigned to but never used

Remove assignment to unused variable `mock_get`

(F841)